src/irr/asset/CGLILoader.cpp

```diff
@@ -0,0 +1,552 @@
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_5x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_5x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_6x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_6x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_6x6: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_6x6_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_8x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_8x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_8x6: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_8x6_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_8x8: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_8x8_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x5: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x5_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x6: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x6_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x8: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x8_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_10x10: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_10x10_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_12x10: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_12x10_KHR
+				case gl::INTERNAL_SRGB8_ALPHA8_ASTC_12x12: return getTranslatedFinalFormat();		//GL_COMPRESSED_SRGB8_ALPHA8_ASTC_12x12_KHR
+
+				case gl::INTERNAL_R8_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_R8_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RG8_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RG8_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGB8_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGB8_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGBA8_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGBA8_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGB10A2_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGB10A2_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_R16_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_R16_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RG16_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RG16_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGB16_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGB16_SSCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGBA16_USCALED_GTC: return getTranslatedFinalFormat();
+				case gl::INTERNAL_RGBA16_SSCALED_GTC: return getTranslatedFinalFormat();
```

Comment:
these look like Vulkan's vertex formats (disallowed for textures) so throw an appropriate error message.